toffy/json_utils_test.py

```diff
@@ -77,3 +77,28 @@ def test_list_moly_fovs(tmpdir):
     pred_moly_fovs = json_utils.list_moly_fovs(tmpdir)
 
     assert np.array_equal(pred_moly_fovs.sort(), moly_fovs.sort())
+
+
+def test_read_json_file():
+    cwd = os.getgwd()
+    # create fake jsons
+    moly_json = {'name': 'bob',
+                 'standardTarget': 'Molybdenum Foil'}
+    json_path = cwd+"/test.json",
+    
+    # write test json
+    with open(json_path, 'w') as jp:
+        json.dump(moly_json, jp)
+
+    # Make sure errors come up when the directory is bad
+     with pytest.raises(ValueError, match='files are the same length'):
```

Comment:
The match here should be the error message that actually gets raised. So what error is validate_paths going to raise for an invalid path? You can run the code without catching the error to check and see what the exact syntax is, then update it here